java/src/main/java/sh/platform/languages/SampleCode.java

```diff
@@ -0,0 +1,45 @@
+package sh.platform.languages;
+
+import java.lang.reflect.Method;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+final class SampleCode {
```

Comment:
I don't quite get what this class does.  Which means it needs docblocks. :smile: 

The static properties confuse me.

Reply:
Done,